index/store/metrics/iterator.go

```diff
@@ -36,6 +42,9 @@ func (i *Iterator) Valid() bool {
 }
 
 func (i *Iterator) Close() error {
+	// record time spent in aggregate
+	i.s.timerIteratorSeek.Update(i.seekTime)
+	i.s.timerIteratorNext.Update(i.nextTime)
```

Comment:
I'm thinking this changes the semantics of the seek time and next time stats.

Previously, it use to be the time of each individual seek() or next().

With this change, it's now different numbers... the totals.